cmd/query/app/otel/otlp_translator_test.go

```diff
@@ -0,0 +1,295 @@
+// Copyright (c) 2021 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package otel
+
+import (
+	"math"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"go.opentelemetry.io/collector/translator/conventions"
+	tracetranslator "go.opentelemetry.io/collector/translator/trace"
+
+	"github.com/jaegertracing/jaeger/model"
+	commonv1 "github.com/jaegertracing/jaeger/proto-gen/otel/common/v1"
+	resourcev1 "github.com/jaegertracing/jaeger/proto-gen/otel/resource/v1"
+	v1 "github.com/jaegertracing/jaeger/proto-gen/otel/trace/v1"
+)
+
+var ts = time.Date(2021, 6, 14, 6, 0, 0, 0, time.UTC)
+
+func TestTranslateSpan(t *testing.T) {
+	traceID := model.NewTraceID(10, 20)
+	traceID2 := model.NewTraceID(10, 20)
```

Comment:
If they have the same ID, why do you need to instances?

Reply:
probably typo